tests/test_processors.py

```diff
@@ -23,8 +32,68 @@ def _get_processor_file(self, processor):
                             'processors',
                             processor.strip() + '.py')
 
+    @staticmethod
+    def _get_first_data(processor, parameters, data_in,
+                        dp_out, data_out, env):
+        '''Returns the first line of data_out as a python object.'''
+        process = subprocess.run([sys.executable, processor, '1',
```

Comment:
I think it would be better to reuse the logic from the parent class instead of rewriting it.